packages/catlog-wasm/src/model.rs

```diff
@@ -177,19 +177,19 @@ impl DblModel {
         })
     }
 
-    #[cfg(test)]
```

Comment:
Thanks Owen, this refactor is great!

I've actually removed/replaced this stuff anyway in #121. What do you think about merging that first and then rebasing this PR?

Reply:
Sounds good!

Reply:
Great, I've gone ahead and merged that one.